pulseaudio-control.bash

```diff
@@ -329,19 +327,20 @@ function usage() {
     echo
     echo "Options: (defaults)"
     echo "    --autosync | --no-autosync            whether to maintain same volume for all programs ($AUTOSYNC)"
-    echo "    --color-mute <rrggbb>                 color in which to format when muted (${MUTED_COLOR:4:-1})"
+    echo "    --color-muted <rrggbb>                color in which to format when muted (${COLOR_MUTED:4:-1})"
     echo "    --notifications | --no-notifications  whether to show notifications when changing sink ($NOTIFICATIONS)"
     echo "    --osd | --no-osd                      whether to display KDE's OSD message ($OSD)"
     echo "    --icon-muted <icon>                   icon to use when muted (none)"
     echo "    --icon-sink <icon>                    icon to use for sink (none)"
     echo "    --icons-volume <icon>[,<icon>...]     icons for volume, from lower to higher (none)"
-    echo "    --volume-max <int>                    maximum volume to which to allow increasing ($MAX_VOL)"
-    echo "    --volume-step <int>                   step size when inc/decrementing volume ($INC)"
+    echo "    --volume-max <int>                    maximum volume to which to allow increasing ($VOLUME_MAX)"
+    echo "    --volume-step <int>                   step size when inc/decrementing volume ($VOLUME_INC)"
     echo "    --sink-blacklist <name>[,<name>...]   sinks to ignore when switching (none)"
-    echo "    --sink-nickname-from <prop>           pacmd property to use for sink name (none)"
+    echo "    --sink-nicknames-from <prop>          pacmd property to use for sink names (none)"
     echo "                                          as listed under the 'properties' key in the output of \`pacmd list-sinks\`"
-    echo "    --sink-nickname <name>:<nick>         nickname to assign to given sink name (may be given multiple times) (none)"
+    echo "    --sink-nickname <name>:<nick>         nickname to assign to given sink name, may be given multiple times (none)"
```

Comment:
Default should probably be the end of the third line now? Looks a little weird mid-sentence. (I realise I introduced that :blush:)

Reply:
I did notice it but wasn't sure what to do. Sure, I'll add the default value at the end.